src/Integration.UnitTests/VsSessionHostTests.cs

```diff
@@ -32,6 +35,11 @@ public void TestInitialize()
             this.serviceProvider = new ConfigurableServiceProvider(assertOnUnexpectedServiceRequest: false);
             this.sonarQubeService = new ConfigurableSonarQubeServiceWrapper();
             this.stepRunner = new ConfigurableProgressStepRunner();
+            this.solutionBinding = new ConfigurableSolutionBinding();
+
+            ConfigurableVsShell shell = new ConfigurableVsShell();
```

Comment:
Putting all of the initialisation for all tests in one places means there is no easy way of telling what is actually required for each test. The tests would be more maintainable if the one test that required the property set it, rather than it being set for all tests - it makes the relationship between the test and required setup clear.